python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -224,10 +224,9 @@ def get_actions(self, data):
 
         manager = sgtk.bootstrap.ToolkitManager()
         manager.base_configuration = self.BASE_CONFIG_URI
-
-        pcs = manager.sort_and_filter_configuration_entities(
```

Comment:
Most likely it does, yeah. Right now there's no use case for it since desktopserver no longer requires it.

internal ticket 42124 lined up for next sprint